src/lib/Behat/PageObject/Page.php

```diff
@@ -43,8 +43,8 @@ public function __construct(UtilityContext $context)
      */
     public function verifyIsLoaded(): void
     {
-        $this->verifyRoute();
         $this->verifyElements();
+        $this->verifyRoute();
```

Comment:
We shouldn't change it here, but override the `verifyIsLoaded` method in UserCreation page.